dashboard/db/migrate/20170224150102_remove_activity_id_from_gallery_activities.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveActivityIdFromGalleryActivities < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
Nope, at least I think it makes sense for the migration to be reversible (though data will be lost (*)).

(*) Actually, not, since the data has been backed up to `s3`. But it would require some manual work to reload it.